### ISSUE TITLE
Stats Dashboard; store information at a glance in one place

### ISSUE THREAD

**Issue by mikejolley** [woocommerce/woocommerce#16974]
The WooCommerce Dashboard will be the first screen seen under the "WooCommerce" menu item in WP admin instead of orders. It will offer at a glance information about current orders, some stats, on-boarding steps, and store notices requiring admin attention.

The current WP dashboard widgets cover:

- Status (sales, top seller, stock alerts, processing order counts)
- Recent reviews

These widgets will be removed and moved/adapted for the new dashboard.

WP Admin also frequently gets spammed with a bunch of WooCommerce/Extension notices and call to actions which can often go ignored. We plan on having a central place to show all notices, not across all of WP Admin, just on our dashboard. A count will be shown in the "WooCommerce" menu showing the count of notices requiring notices.

Full list of ideas for items to include on the dashboard:

- Admin notices, collected and included only on this screen.
- On-boarding steps i.e. Create your first product, setup shipping etc. These will appear and be hidden once performed or dismissed. They will appear like notices.
- At a glance stats i.e. for sales
- Orders requiring action (list? count?)
- Products requiring action e.g. low stock levels (list? count?)
- New extensions (this is mainly for marketing)
- WooCommerce news and meetups
- Tips?
- Status report warnings?

This issue needs design input before we start any sort of development.

**Comment by DavidAnderson684** [comment]
If people ignore WC notices on the admin dashboard, how likely are they to notice the notice count and then make a special visit to the WC dashboard? Or to be polling the WC dashboard for any other reason (and then to notice the notices among the other things present, when they currently ignore them in that situation)?

For people who do read dashboard notices, this seems like a step back  ... we'll have to remember to poll a second page to get the WC notices and/or keep an eye on the notice count, whereas before we just had to read dashboard notices. Personally, if there's a filter to make them still appear on the main dashboard, I'll turn that straight on, as I'd find adding one more "message centre that needs polling" to the list to be undesirable.

**Comment by mikejolley** [comment]
Well the other argument for this is that messages related specifically to WC should not be strewn throughout WP admin. 

There will be an always visible count, so those aware of notices will know when there are some requiring attention. Just like plugin updates.

**Comment by addedlovely** [comment]
It's not unusual to have various plugin notifications bleed into other admin pages as @mikejolley says. I have one plugin which is licensed, but it shows an renew license on every admin page. The notifications circle thing is nice, but again might end up getting misused to grab attention / advertise, I've dismissed the Yoast SEO review nag notification too many times now.

I'm not a plugin developer, but is there a standard api beyond admin_notices for showing these notices? Perhaps a good first step would be to provide a standard function for the plugin developers. Then can standardise the display and behavouir. 

Regarding the dashboard, for a broader perspective, the Shopify dashboard is quite impressive, with sales insights, apps and tips presented in a rolling blog format. Some of the 'insight' posts:

* These returning customers haven’t made a purchase recently
* These returning customers haven’t made a purchase recently
* In the last 30 days 7.5% of your abandoned checkout recovery emails were successful  
* Since store launch - Your store is getting a lot of traffic (visitor metrics)
* In the last 14 days - These products were viewed most often (popular product metrics)
* Some of your visitors can’t find what they're looking for (empty search result metrics)

Obviously quite a few of these are stats WooCommerce might not be able to provide out of the box, but that format / design could work quite nicely for plugin developers to hook into. It also feels less needy than yet another red notification box to dismiss.

**Comment by claudiulodro** [comment]
Feature plugin (WIP) for this is available at https://github.com/woocommerce/wc-admin at the moment and will be merged into core in a major release next year.
